### ISSUE TITLE
Spotless neither checks nor fixes files in `src/test/java`

### ISSUE THREAD

**Issue by J-N-K** [openhab/openhab-core#3515]
Found in #3510 

The configuration is correctly imported and changed files are properly identified, but then they are reported as "clean" even if they are not. This could be a configuration issue at our end or n external bug.

**Comment by J-N-K** [comment]
It's even more confusing.

```
public class   InputStreamCacheWrapperTest {
```

get fixed to

```
public class   InputStreamCacheWrapperTest {
```

while in the same file 

```
@Test
    public void           cacheWrapperStreamReadTest() throws IOException {
        when(cacheEntry.read(0, 1)).thenReturn(new byte[] { 1 });
```

is accepted and not changed. So it looks like the file gets processed, but no all checks are applied. If I do the same to files under `/src/main/java`, the extra-spaces are correctly removed.

**Comment by wborn** [comment]
It seems to check the files again when using 4.25 here:  https://github.com/openhab/openhab-core/blob/4851bfbcae59861ce5a1bb4a2e8d08a456a60cbc/pom.xml#L81

Doesn't seem to be fixed when using newer versions there.

**Comment by J-N-K** [comment]
I think that 4.26 is needed to fix the other spotless issues with method calls in instanceof.

**Comment by holgerfriedrich** [comment]
4.27 seems to be available, (together with spotless 2.37.0....) I cannot check if it fixes the issue above, as I am not using Eclipse.

Changelog is here: https://github.com/diffplug/spotless/blob/main/plugin-maven/CHANGES.md
